SIPS/sip-32.md

```diff
@@ -47,19 +47,12 @@ pre-installed.
 #### Parameters
 
 - `event` - Event object to be tracked.
-  - `category: string` - The category to associate the event.
-
-  - `event: string` - The name of the event to track.
+  - `name: string` - The name of the event to track.
 
   - `properties: Record<string, Json>` - (**Optional**) Custom values to track.
     The client MUST enforce that all keys in this object are in the
     `snake_case` format.
 
-  - `sensitiveProperties: Record<string, Json>` - (**Optional**) Sensitive
```

Comment:
As far as I can tell we cannot remove this

Reply:
Reverting this until a decision is made